src/applications/auth/containers/AuthApp.jsx

```diff
@@ -133,10 +133,9 @@ export class AuthApp extends React.Component {
     const returnUrl = sessionStorage.getItem(authnSettings.RETURN_URL) || '';
     sessionStorage.removeItem(authnSettings.RETURN_URL);
 
-    const postAuthUrl =
-      returnUrl.includes('?next=') && !environment.isProduction()
-        ? appendQuery(returnUrl, 'postLogin=true')
-        : returnUrl;
+    const postAuthUrl = !environment.isProduction()
```

Comment:
Leaving the `postAuthUrl` so we can test solely on staging. Once we confirm this works on staging, I will append the param directly to the `returnUrl` and remove `postAuthUrl`.

Reply:
What happens if `returnUrl` is blank?

Reply:
That's a good point, it would still append the query to the blank `returnUrl`. If it's blank it should return the blank string for `postAuthUrl`. I'll update that.